model/src/main/java/org/mskcc/cmo/metadb/model/MetaDbRequest.java

```diff
@@ -166,6 +166,16 @@ public void addMetaDbSampleList(MetaDbSample metaDbSample) {
         }
         metaDbSampleList.add(metaDbSample);
     }
+    
```

Comment:
same comment here about the request. we should. just override the `equals()` and `hashCode` methods and _then_ worry about the list comparisons